README.md

```diff
@@ -28,6 +28,11 @@ and compiled with Go 1.4. Earlier FreeBSD releases are not supported.
 
 Getting Started
 ---------------
+### VM with vagrant
```

Comment:
Please add that Ansible needs to be installed on the Vagrant host. I've just had to figure that one out, as I'm not familiar with Ansible and am used to Chef/Puppet paradigm where the configuration management runs on the configured machine, so I expected that the provisioner will take care of installing Ansible in the VM.